### DISCUSSION TITLE
Query example

### DISCUSSION THREAD

**Discussion by mjfelis** [trustoverip/tswg-trust-registry-protocol#63] [Q&A] [answered]
Dear trqp team,

I'm trying to figure out what would be a proper example query, if I want to make sure `did:issuer:678`  is permitted to issue a credential of a given schema `https://example.com/credentials/3734`.

It's not clear to me at the moment how do I securely "bind" a schema to an authorization in a Trust Registry.

Any feedback would be appreciated.

BR,
Fabrice

**VC**:

```json
{
   "@context":[
      "https://www.w3.org/ns/credentials/v2",
      "https://www.w3.org/ns/credentials/examples/v2"
   ],
   "id":"https://example.com/credentials/3733",
   "type":[
      "VerifiableCredential",
      "ExampleEmailCredential"
   ],
   "issuer":"did:issuer:678",
   "issuanceDate":"2010-01-01T19:23:24Z",
   "credentialSubject":{
      "id":"did:example:ebfeb1f712ebc6f1c276e12ec21",
      "emailAddress":"subject@example.com"
   },
   "credentialSchema":{
      "id":"https://example.com/credentials/3734",
      "type":"JsonSchemaCredential"
   }
}
```

Issuer DID Document:

{
   "@context":[
      "https://w3id.org/did/v1",
      "https://w3id.org/security/suites/ed25519-2018/v1",
      "https://w3id.org/security/suites/x25519-2019/v1"
   ],
   "id":"did:issuer:678",
   "service":[
      {
         "id":"did:issuer:678#tr",
         "type":"TrustRegistry",
         "serviceEndpoint":[
            {
               "type":"Trust Registry",
               "profile":"https://trustoverip.org/profiles/trp/v2",
               "uri":"https://my-tr-service/"
            }
         ]
      }
   ]
}

**Comment by darrellodonnell** [comment, marked as answer]
for your credential type and action, you'd want to create a simple string. Something like "mycredtype.issue" or "mycredtype.fullproof" for an Issuer and Verifier respectively.
